### PR TITLE
Add program options for database connection

### PR DESCRIPTION
**Pull request by Pyeroh**
Not related to an existing issue. Should I open one ?

With this PR, one should be able to call the program with connection informations, instead of providing them in the config file.

### CODE REVIEW

File: src/main/java/com/rolfje/anonimatron/Anonimatron.java [hunk 1]
```diff
@@ -76,9 +91,18 @@ public static void main(String[] args) throws Exception {
...
 		Configuration config = Configuration.readFromFile(configFile);
+		if (jdbcurl != null) {
+			config.setJdbcurl(jdbcurl);
+		}
+		if (userid != null) {
+			config.setUserid(userid);
+		}
+		if (password != null) {
+			config.setPassword(password);
+		}
```
Comment (realrolfje) [inline src/main/java/com/rolfje/anonimatron/Anonimatron.java]: Neat! That would also enable users to create a config file without having to have a password in it. Overriding just the password on the command line. Thanks!

**Comment by realrolfje** [comment]
Great addition, thanks! I hope to merge it somewhere this week. Nice thing to mention in the documentation is that with this PR, people can have a config file without the database password, which is a really nice feature.
If you like you can make an issue for it so that we can close it with this PR, but for me that is not absolutely required.

**Comment by Pyeroh** [comment]
Thanks for your review ! I'll add this bit in the readme :)

**Comment by realrolfje** [comment]
Merged into https://github.com/realrolfje/anonimatron/tree/feature/connection-info-in-args and I will do some more cleanup after that. Will be part of next release too.

**Pull request merged**